_data/rishavjaiss/rishavjaiss.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Rishav Jaiswal
+institution: MCKV Institute Of Engineering
+profile_pic: profile.png
+quote: work untill My dream company becomes My company.

```

Comment:
Typo in the quote, change `untill` to `until`.

Reply:
Hey @Mohammed-Shoaib , thanks for pointing that out i overlooked it. It's done now.